fl4health/clients/basic_client.py

```diff
@@ -541,6 +581,8 @@ def setup_client(self, config: Config) -> None:
 
         self.wandb_reporter = ClientWandBReporter.from_config(self.client_name, config)
 
+        self.metrics_reporter.add_to_metrics({"type": "client", "initialized": datetime.datetime.now()})
```

Comment:
It might be useful to log the "client_name" in the metrics somewhere (i.e. the unique hash)? However, if you don't think that would be helpful, no need.

Reply:
I don't think it's necessary. For the file name, when it's saved, it's already given with a UUID if an id is not provided.

Reply:
Fair enough. Wasn't sure if it would be useful for mapping the metrics back to a specific client.